frontend/Flup/.gitignore

```diff
@@ -0,0 +1,44 @@
+# Miscellaneous
```

Comment:
```suggestion
# Platform specific folders
android/
ios/
macos/
web/
windows

# Miscellaneous
```

Reply:
Nevermind. These files are created when `flutter create` is run and not auto-generated when building.
https://github.com/flutter/flutter/issues/12639
They should stay in the repo